jsonnet/kube-prometheus/addons/pyrra.libsonnet

```diff
@@ -22,6 +22,11 @@
     namespace:: error 'must provide namespace',
     version:: error 'must provide version',
     image: error 'must provide image',
+    resources: {
+      // TODO(paulfantom): configure limits when we have more operational data
+      // limits: { cpu: '100m', memory: '100Mi' },
+      requests: { cpu: '4m', memory: '100Mi' },
```

Comment:
This might get the API throttled just a bit too much. Currently, we convert the time series into another format.

Reply:
Since the most important part of this PR is to allow easy customization of those resources and not figure out the best possible values, I've removed cpu resource requirements and left only memory ones. We can change this later on.